lib/utils/showHtmlToast.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+// eslint-disable-next-line import/prefer-default-export
+export function showHtmlToast(messageId, messageType = 'success', values = {}) {
```

Comment:
I would prefer to have a config object, rather than separate optional attributes, it's easier to extend in future
`showHtmlToast(messageId, config = {messageType: 'success'})`
`message: <SafeHTMLMessage id={messageId} values={config.values || {}} />,`

Reply:
Done